ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout.kt

```diff
@@ -0,0 +1,94 @@
+package org.cru.godtools.tutorial
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+//import org.cru.godtools.R
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import java.nio.file.Files.size
+
+@Preview(showBackground = true)
+@Composable
+internal fun TipsLayout() = GodToolsTheme() {
+    Column(
+        modifier = Modifier.fillMaxSize().padding(top = 100.dp),
+
+        //verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ){
+        Spacer(modifier = Modifier.height(100.dp))
+        val composition by rememberLottieComposition(
+            LottieCompositionSpec
+                .RawRes(org.cru.godtools.tutorial.R.raw.anim_tutorial_tips_people)
+        )
+        var isPlaying by remember {
+            mutableStateOf(true)
+        }
```

Comment:
we won't be toggling whether the animation is playing or not, it will always be playing.

so, there is no need to create this state variable, you can just directly pass in true for `isPlaying` in the `animateLottieCompositionAsState` method.

Reply:
it actually looks like the default value for `isPlaying` is `true`, so you wouldn't need to pass in that argument to `animateLottieCompositionAsState` at all